getting-started-service-registry/README.adoc

```diff
@@ -0,0 +1,197 @@
++
+--
+[.screencapture]
+.Guided steps to define topic details
+image::upload-schema.png[Image of form to upload a schema]
+
+* *Group*: Enter a unique group name such as `my-org`, to organize and put this artifact into a named collection. Each group contains a logically related set of schemas or API designs, typically managed by a single entity, belonging to a particular application or organization.
++
+NOTE:  Specifying a group is optional when using the Service Registry web console, where a `default` group is automatically created. When using the REST API or Maven plug-in, you can specify the `default` group in the API path if you do not want to create a unique group.
++
+* *ID*: Set a unique for this group artifact ID such as `my-ID`. If you do not specify a unique artifact ID, Service Registry generates one automatically as a UUID.
+* *Type*: Use the default Auto-Detect setting to automatically detect the artifact type, or select the artifact type from the drop-down, for example, Avro Schema or OpenAPI.
+* *Artifact*: Artifact: Drag and drop or click Browse to upload a file. For this example copy and paste the following AVRO schema:
+`{
+"type": "record",
+"namespace": "com.example",
+"name": "FullName",
+"fields": [
+{ "name": "first", "type": "string" },
+{ "name": "last", "type": "string" }
+]}`.
+
+After you complete the schema creation, the new artifact details are displayed.
+
+* *Info*: Displays the artifact name, group, description, lifecycle status, when created, and last modified. Click the Edit Artifact Metadata pencil icon to edit the artifact name and description or add labels, and click Download to download the artifact file locally.
+* *Content*: Displays a read-only view of the full artifact content.
+* *Documentation*: (OpenAPI only): Displays automatically-generated REST API documentation.
+* *Content Rules*: Also displays artifact Content Rules that you can enable and configure. You can configure a Validity Rule or Compatibility Rule, by selecting the appropriate rule configuration from the drop-down. For more details, see here (TODO add link).
+
+You can now start serialazing and deserializing messages using this schema from Kafka applications.
+--
+
+. On the right side of the Artifact Details, click Upload new version to add a new artifact version.
+. Use the trash next to Upload new version to delete the artifact.
++
+IMPORTANT: Deleting an artifact deletes the artifact and all of its versions, and cannot be undone. Artifact versions are immutable and cannot be deleted individually.
+.
+
+.Verification
+ifdef::qs[]
+* Is the new schema in the Artifacts table?
+endif::[]
+ifndef::qs[]
+* Verify that the new schema is listed in the Artifacts table.
+endif::[]
+
+[role="_additional-resources"]
+== Additional resources
+* https://kafka.apache.org/081/documentation.html#configuration[Configuration^] in Kafka
+(TODO) add link the RHOSR docs
```

Comment:
Link to the RHOSR docs will be https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry